src/styles/main.scss

```diff
@@ -1,7 +1,32 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import "./utils/mixins";
+@import "./utils/reset";
+@import "./utils/variables";
+
+@import "./blocks/header";
+@import "./blocks/page";
+@import "./blocks/nav";
+@import "./blocks/menu";
+@import "./blocks/strategic";
+@import "./blocks/slider";
+@import "./blocks/main";
+@import "./blocks/about-us";
+@import "./blocks/our-expertise";
+@import "./blocks/services";
+@import "./blocks/process";
+@import "./blocks/testimonials";
+@import "./blocks/vision";
+@import "./blocks/send-us-a-message";
+@import "./blocks/contact-us";
+@import "./blocks/footer";
 
```

Comment:
```
* {
  box-sizing: border-box;
}
```

please add this global styling here and remove `box-sizing: border-box;` from other places in project